nifi-nar-bundles/nifi-standard-bundle/nifi-standard-processors/src/main/java/org/apache/nifi/processors/standard/GenerateTableFetch.java

```diff
@@ -115,20 +128,36 @@ public GenerateTableFetch() {
 
     @OnScheduled
     public void setup(final ProcessContext context) {
+        // The processor is invalid if there is an incoming connection and max-value columns are defined
+        if (context.getProperty(MAX_VALUE_COLUMN_NAMES).isSet() && context.hasIncomingConnection()) {
+            throw new ProcessException("If an incoming connection is supplied, no max-value column names may be specified");
```

Comment:
The original concern is for tables that don't share the same name for max-value column. Now that you mention it, I agree that we could allow for a static string or EL result (maybe without using an incoming flow file?), and just document that all the specified tables must contain the max-value columns.  This makes it consistent with Column Names, which currently doesn't have that check but has the same requirement. I will add doc to both and update the logic, thanks!